### ISSUE TITLE
Instruction gif is confusing

### ISSUE THREAD

**Issue by thomasmorgan** [Ruben222/Social-learning-spite#4]
I know how the experiment works, but found it hard to follow, I suspect ppts will find it harder.

**Comment by Rowatsonn** [comment]
Ah that's a shame, I was quite proud of the GIF this time. Do you think it has too much information on it? Or is there something generally that I could change to make it clearer?

**Comment by thomasmorgan** [comment]
I think there's not enough info. There are too few frames and numbers appear with no clear indication of what they are. Also, the big green "X" for the pot multiplier looks too much like a "NO" or "forbidden" sign. I'd add more intermediary frames, perhaps with some text labels to guide the viewer.
